manifest.json

```diff
@@ -23,7 +23,7 @@
         "email": "tobias.ollive@oiseauroch.fr"
     }],
     "requirements": {
-        "yunohost": ">= 4.2.0"
+        "yunohost": ">= 11.0.9"
```

Comment:
Thanks for all the work @ericgaspar  of maintaining good practices. I'm wondering about this version requirements. Did you do change that necessite this version ? The thing is because there is a major version upgrade, I would have prefer to let previous requirement version to keep the app usable for people that didn't upgrade to new major version.